WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,32 +132,37 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
 	
 	allowed_to_hold = {
 		is_adult = yes
 		is_ruler = no
 		is_female = no
-		trait = poet
+		OR = {
+			trait = poet
+			learning = 14
+		}
```

Comment:
Yes, but someone who has no fun in writing poetry isn't fit for this office. Your case is that we should add more flavour around this, but you confirm me in my view that what you suggest isn't a solution to anything nor desired.